Aiolos/Aiolos/Sources/Panel.swift

```diff
@@ -33,6 +33,7 @@ public final class Panel: UIViewController {
     @objc public var isVisible: Bool { return self.parent != nil && self.animator.isTransitioningFromParent == false }
     public weak var sizeDelegate: PanelSizeDelegate?
     public weak var animationDelegate: PanelAnimationDelegate?
+    public weak var gestureRecognizerDelegate: UIGestureRecognizerDelegate?
```

Comment:
Great idea to have this in its own delegate, I'd just change the name a bit for simplicity reasons
```suggestion
    public weak var gestureDelegate: UIGestureRecognizerDelegate?
```